packages/javascript/bh-shared-ui/src/components/AssetGroupFilters/AssetGroupFilters.tsx

```diff
@@ -30,46 +29,45 @@ import {
     Paper,
     Select,
 } from '@mui/material';
-import createStyles from '@mui/styles/createStyles';
 import makeStyles from '@mui/styles/makeStyles';
 import { Theme } from '@mui/material/styles';
+import NodeIcon from '../NodeIcon';
+import { AssetGroupMemberCounts } from 'js-client-library';
 
 export const FILTERABLE_PARAMS: Array<keyof Pick<AssetGroupMemberParams, 'primary_kind' | 'custom_member'>> = [
     'primary_kind',
     'custom_member',
 ];
 
-const useStyles = makeStyles((theme: Theme) =>
-    createStyles({
-        formControl: {
-            display: 'block',
-        },
-        activeFilters: {
-            '& button.expand-filters': {
-                fontWeight: 'bolder',
-                '& span': {
-                    visibility: 'visible',
-                },
+const useStyles = makeStyles((theme: Theme) => ({
+    formControl: {
+        display: 'block',
+    },
+    activeFilters: {
+        '& button.expand-filters': {
+            fontWeight: 'bolder',
+            '& span': {
+                visibility: 'visible',
             },
         },
-        activeFiltersDot: {
-            width: '6px',
-            height: '6px',
-            borderRadius: '100%',
-            backgroundColor: theme.palette.primary.main,
-            alignSelf: 'baseline',
-            visibility: 'hidden',
-        },
-    })
-);
+    },
+    activeFiltersDot: {
+        width: '6px',
+        height: '6px',
+        borderRadius: '100%',
+        backgroundColor: theme.palette.primary.main,
+        alignSelf: 'baseline',
+        visibility: 'hidden',
+    },
+}));
 
 interface Props {
     filterParams: AssetGroupMemberParams;
     handleFilterChange: (key: (typeof FILTERABLE_PARAMS)[number], value: string) => void;
-    availableNodeKinds: Array<ActiveDirectoryNodeKind | AzureNodeKind>;
+    memberCounts: AssetGroupMemberCounts | undefined;
 }
 
-const AssetGroupFilters: FC<Props> = ({ filterParams, handleFilterChange, availableNodeKinds }) => {
+const AssetGroupFilters: FC<Props> = ({ filterParams, handleFilterChange, memberCounts }) => {
```

Comment:
Would it be easier to make this an optional param with default value {}?

Reply:
Good suggestion! We will need a default value of `{ counts: {}}` but I think that is cleaner than the fallback below! Then if we add more logic around memberCounts.counts we dont always need to fallback.

Updating!